SpatialGDK/Source/Public/Interop/SpatialPlayerSpawner.h

```diff
@@ -33,6 +33,6 @@ class SPATIALGDK_API USpatialPlayerSpawner : public UObject
 	UPROPERTY()
 	USpatialNetDriver* NetDriver;
 	
-	FTimerManager* TimerManager;
+	class FTimerManager* TimerManager;
```

Comment:
What do you think about doing a forward declaration of `FTimerManager` at the top of the file?

Reply:
That would also fix the problem. Only concern is that it might mask the issue in other files in a Unity build, but I guess it's a cleaner solution. Will change.